.github/00-bug-performance-issue.md

```diff
@@ -0,0 +1,31 @@
+---
+name: Bug/Performance Issue
+about: Use this template for reporting a bug or a performance issue.
+
+---
+
+<em>Please make sure that this is a bug. As per our [GitHub Policy](https://github.com/tensorflow/tensorflow/blob/master/ISSUES.md), we only address code/doc bugs, performance issues, feature requests and build/installation issues on GitHub. tag:bug_template</em>
+
+Consider Stack Overflow for getting support using TensorBoard - they have a larger community with better searchability:
```

Comment:
We should not suggest it right away. Let's wait until we have StackOverflow support

Reply:
Done